projects/plugins/jetpack/extensions/blocks/subscriptions/utils.js

```diff
@@ -20,9 +20,10 @@ export const encodeValueForShortcodeAttribute = value => {
 		.replace( /\u200b/g, '&#x200b;' );
 };
 
-export const getPaidPlanLink = hasNewsletterPlans => {
+export const getPaidPlanLink = () => {
 	const link = 'https://wordpress.com/earn/payments-plans/' + location.hostname;
-	return hasNewsletterPlans ? link : link + '#add-newsletter-payment-plan';
+	// Add the flag to enable newsletter
+	return link + '#add-newsletter-payment-plan';
```

Comment:
We could just make this one line

Reply:
[This ](https://github.com/Automattic/wp-calypso/pull/70368) fixes this.

Reply:
> [Automattic/wp-calypso#7036](https://github.com/Automattic/wp-calypso/pull/7036) fixes this.

This link is 6 years old ;)

Reply:
It's been a long day. I meant https://github.com/Automattic/wp-calypso/pull/70368